package.json

```diff
@@ -12,7 +12,8 @@
     "@reduxjs/toolkit": "^1.9.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-redux": "^8.0.5"
+    "react-redux": "^8.0.5",
+    "react-router-dom": "^6.8.1"
   },
   "devDependencies": {
     "@types/react": "^18.0.26",
```

Comment:
We are using yarn in this project to manage dependencies. There was no need to install them using npm.
Kindly delete package-lock.json.